src/main/frontend/src/pages/Shop.js

```diff
@@ -23,26 +23,27 @@ function Shop() {
   const { categoryId } = useParams();
   const options = [
     {
-      value: "default",
+      value: { orderBy: SORT_BY.DEFAULT, direction: ORDER.ASC },
       name: "Default Sorting",
     },
     {
-      value: "popularity",
-      name: "Sort by Popularit",
+      value: { orderBy: SORT_BY.NEWNESS, direction: ORDER.ASC},
+      name: "Sort by Newness",
     },
     {
-      value: "rating",
-      name: "Sort by Rating",
+      value: { orderBy: SORT_BY.TIME_LEFT, direction: ORDER.ASC },
+      name: "Sort by Time Left",
```

Comment:
As per ticket, this should only be Time left, without `sort by` part. 
Applies to other options as well